crates/pallet-balanced-currency-swap-bridges-initializer/src/upgrade_init.rs

```diff
@@ -22,7 +22,14 @@ pub fn on_runtime_upgrade<T: Config>() -> Weight {
             }
         }
 
-        <InitializerVersion<T>>::put(CURRENT_BRIDGES_INITIALIZER_VERSION);
+        <InitializerVersion<T>>::put(T::InitializerVersion::get());
+    } else if T::IsBalancedCheckRequiredOnRuntimeUpgrade::get() {
```

Comment:
Doesn't look like this is the logic we have discussed, so we'll need to discuss this again.

Reply:
https://github.com/humanode-network/humanode/pull/785/commits/73c28576205966c2e867eebe0280a511778dc8f8